dev/templates/packages.R

```diff
@@ -0,0 +1,7 @@
+# R packages for RNA sequence workshop part 1 
```

Comment:
Can you make this comment more general, for instance
`# Use this script to install packages via CRAN, for example:`